backend/app/tests/ui/pages/home.py

```diff
@@ -23,16 +52,74 @@ def is_create_new_pdf_button_displayed(self):
     def create_pdf_modal_is_open(self):
         return self.is_element_displayed(*self._pdf_job_form_modal_locator)
 
+    @property
+    def create_pdf_modal_is_closed(self):
+        return self.is_element_displayed(*self._create_new_pdf_button_locator)
+
     def click_create_new_pdf_button(self):
         self.find_element(*self._create_new_pdf_button_locator).click()
         self.wait.until(lambda _: self.create_pdf_modal_is_open)
-        return self.CreatePDFModal(self, self.find_element(*self._pdf_job_form_modal_locator))
+        return self.CreatePDFModal(
+            self, self.find_element(*self._pdf_job_form_modal_locator)
+        )
+
+    def click_previous_page_button(self):
+        self.find_element(*self._previous_page_button_locator).click()
+        return self
+
+    def click_next_page_button(self):
+        self.find_element(*self._next_page_button_locator).click()
+        return self
+
+    def click_go_button(self):
+        self.find_element(*self._go_button_locator).click()
+        return self
+
+    @property
+    def page_field(self):
+        return self.find_element(*self._page_field_locator)
+
+    def fill_in_page(self, value):
+        self.page_field.send_keys(value)
+        return self
+
+    def no_data_available(self):
+        return self.find_element(*self._no_data_available)
 
     class CreatePDFModal(Region):
         _modal_cancel_button_locator = (
-            By.CSS_SELECTOR, '#app > div.v-dialog__content.v-dialog__content--active > div > div > div.v-card__actions > button:nth-child(2) > span'
+            By.CSS_SELECTOR,
+            "#app > div.v-dialog__content.v-dialog__content--active "
+            "> div > div > "
+            "div.v-card__actions > button:nth-child(2) > span ",
+        )
+        _modal_create_button_locator = (
+            By.CSS_SELECTOR,
+            "#app > div.v-dialog__content.v-dialog__content--active "
+            "> div > div > div.v-card__actions > "
+            "button:nth-child(3) > span ",
         )
 
         def click_cancel_button(self):
             self.find_element(*self._modal_cancel_button_locator).click()
             return self
+
+        def click_create_button(self):
+            self.find_element(*self._modal_create_button_locator).click()
+            return self
+
+        def fill_collection_id_field(self, value):
```

Comment:
I like the use of the TAB key here (lots of potential for accessibility testing) but it's a bit misleading, no? Let's say I'm writing a test and for some reason I want to jump ahead to `fill_version_field` and the focus of the element is not the one previous to it. The name of the method gives me the impression I'll be able to fill the version field but actually all I'm doing is hitting tab and entering a value. In other words, I will not actually be filling the version field.

I think it would be better to change this code to select the proper input box and then send the value rather than tabbing through. I do thing tabbing through could be a separate test that compares you do things in the right order for the future.